project/api/models.py

```diff
@@ -1,27 +1,52 @@
 from django.core.exceptions import ValidationError
 from django.contrib.gis.db import models
 from simple_history.models import HistoricalRecords
+from colorfield.fields import ColorField
 
 # Create your models here.
 class Nation(models.Model):
-    name = models.SlugField(max_length=20)
-    local_name = models.CharField(max_length=30)
-    wikipedia = models.URLField()
+    """
+    Cultural/governmental entity. Serves as foreign key for most Territories
+    """
+    name = models.CharField(max_length=20,
+                            help_text="Canonical name, should not include any epithets, must be unique",
+                            unique=True)
+    color = ColorField(default="#FF0000",
+                       help_text="Color to display on map",
+                       unique=True)
     history = HistoricalRecords()
 
+    #Flavor fields
+    aliases = models.TextField(help_text="CSV of alternative names this state may be known by",
+                               blank=True)
+    description = models.TextField(help_text="Flavor text, brief history, etc.",
+                                   blank=True)
+    wikipedia = models.URLField(help_text="Link to the Wikipedia article for this nation",
+                                blank=True)
+
+    #History fields
+
+    # Foreign key to auth.User which will be updated every time the model is changed,
+    # and is this stored in history as the user to update a specific revision
+    # Consider other metadata such as DateTime for the revision (may be handled by django-simple-history)
+    # TODO: implement this
+
     def __str__(self):
         return self.name
 
 class Territory(models.Model):
-
+    """
+    Defines the borders and controlled territories associated with a Nation.
+    """
     class Meta:
         verbose_name_plural = "territories"
 
-    start_date = models.DateField()
-    end_date = models.DateField()
-    geo = models.GeometryField()
+    start_date = models.DateField(help_text="When this border takes effect")
+    end_date = models.DateField(help_text="When this border ceases to exist")
+    geo = models.MultiPolygonField()
```

Comment:
To be future-proof, I'd put a `FeatureField` here. We then define the geometry (Polygon or MultiPolygon), and optionally we could pass some properties to the feature (instead of creating a new field on the model).

Reply:
Slightly confused, what use cases do you have in mind for something that would be passed as properties rather than a new field?

Reply:
I was thinking about styling properties? The color of the territory is the color of the nation, but I was imagining a striped territory, for whatever reason (like the ones we saw in history books at school).

Though now I'm having 2nd thoughts, it's better let this whole territory model behave like a Feature, and the geo field would be the geometry field, like currently done. If we do add properties, let's add them properly as fields, it'll be easier to track.

Reply:
Another question: Why MultiPolygonField instead of GeometryField? I'd go for GeometryField so that we can choose if it's a MultiPolygon or a Polygon

Reply:
Geometry objects also allow for points and lines which I don't think would be applicable. It is possible for a MultiPolygon field to just contain one Polygon however, and I believe the field accepts GeoJSON for each as input

Reply:
As for your earlier comment I agree, I think that would be best supplied as an additional field on the Territory model, maybe a type field, or do something with the Diplomatic Relation field

Reply:
Sounds good for the additional fields!

Still not 100% convinced about the MultiPolygon. I think it's more semantically correct for the API to return a Polygon when the territory is an actual Polygon. And maybe add a validation on the field to disallow all other Geometries. Feels slightly less "hacky".

Reply:
Fair point, I'll implement this.